src/sentry/runner/commands/configoptions.py

```diff
@@ -135,19 +147,24 @@ def configoptions(ctx, dry_run: bool, file: Optional[str], hide_drift: bool) ->
     ctx.obj["options_to_update"] = options_to_update
 
     drifted_options = set()
+
+    json_data = {"dry-run": dry_run, "updated_options": [], "drifted_options": []}
```

Comment:
While this approach: adding the slack specific code in the script as it is today would work, I would advise against and adding an abstraction instead in order to separate the responsibilities between the options update code and the presentation code as the presentation code is becoming less straightforward.

This can be done by:
- create a common interface between the different presentation mechanism (console and slack), this can contain methods that correspond to the logical functionality they solve:
```
class OptionUpdatePresenter(ABC):
   def drifted_option(....)

   def skipped_option(....)

   def error(....)

   def write(....)
```

2) Have two implementations: one for the console and one for slack 

3) Have another implementation that couples the two. This third implementation reads sentry settings to see if the slack webhook i configured (it will mostly be not configured in self hosted) and decides whether to instantiate the slack notifier.

So the configoptions code that is responsible to manage the updates does not care how data is presented. The responsibility of the presentation falls entirely on the Presenter. 
Then The presenter can be tested in isolation and the presenter is not really needed when testing the configoptions update.